switch/libode/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+
+# Maintainer: davidgfnet <david@davidgf.net>
+pkgname=switch-libode
+pkgver=0.16
+pkgrel=1
+pkgdesc='ODE (Open Dynamics Engine): real-time physics simulation featuring advanced collision with friction and joint types. Useful for games and simulation tools. (for Nintendo Switch homebrew development)'
```

Comment:
Description here is a touch unwieldy - arch favours short descriptions that don't include package names, see https://wiki.archlinux.org/index.php/PKGBUILD#pkgdesc 

The [arch package here](https://www.archlinux.org/packages/community/x86_64/ode/) uses just "High performance library for simulating rigid body dynamics" which is probably fine. I know I've used "For X homebrew development" in quite a few other places but I think that's superfluous given that we already know what console we're targeting from the package prefix.